### PR TITLE
Bug 1182509 - [Contacts][NGA] Open contacts_matching with window.open

### PR DESCRIPTION
**Pull request by borjasalguero**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8c298c68bfc17a1237dc0c3339f3c6d359078246

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9ac0cc1b94dd7f30d36708e7e059a06943bda6f2

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=42b93b83e33f9123f29d2e82804b41b896134c80

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6c9fbaa3dc2aa616facc3de4656c083d693875cd

File: apps/communications/contacts/js/match_service.js [hunk 1]
```diff
@@ -1,45 +1,63 @@
...
-      '/contacts/views/matching/matching_contacts.html?contactId=' + contactId;
+    var url =
+      '/contacts/views/matching/matching_contacts.html?contactId=' +
+      contactId;
+    openWindow(url);
   };
 
   matchService.showDuplicateContacts = function() {
-    extensionFrame.src = currentURI = 
+    var url =
```
Comment (arcturus) [inline apps/communications/contacts/js/match_service.js]: Why we don't just `openWindow('....');`

**Pull request merged**